gemini/use-cases/retrieval-augmented-generation/intra_knowledge_qna.ipynb

```diff
@@ -0,0 +1,832 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    You are helpful AI assistant. You're tasked to answer the question given below, but only based on the context provided.
</pre><p>You are a helpful AI Assistant</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/483/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Changed.</p>